lib/grp.gi

```diff
@@ -4453,9 +4453,14 @@ end);
 ##  anyhow,you should compute it before computing the maximal normal
 ##  subgroups.
 ##
+##  *Note* that for abelian and solvable groups the maximal normal subgroups
+##  can be computed very quickly. Thus if you suspect your group to be
+##  abelian or solvable, then check it before computing the maximal normal
+##  subgroups.
+##
```

Comment:
This is not documented in the documentation, and I  wonder whether this would be appropriate (actually both points: If you can get your hands on a character table, or if the group is abelian or solvable `MaximalNormalSubgroups` can be computed more effciently)?